src/os/bluestore/BlueStore.cc

```diff
@@ -5397,6 +5397,23 @@ void *BlueStore::MempoolThread::entry()
     interval_stats_trim = false;
 
     store->refresh_perf_counters();
+    uint64_t period = store->cct->_conf.get_val<uint64_t>("bluestore_fragmentation_check_period");
+    if (period != 0 && store->alloc) {
+      auto now = mono_clock::now();
+      timespan elapsed = now - last_fragmentation_check;
+      if (elapsed > make_timespan(period)) {
+        last_fragmentation_check = now;
+        double score;
+        score = store->alloc->get_fragmentation_score();
+        store->logger->set(l_bluestore_fragmentation, score * 1e6);
```

Comment:
@aclamk Was it intentional to make the fragmentation score a thousand times larger than before?

Prior to the PR, the fragmentation score returned by the `ceph_bluestore_fragmentation_micros` metric was simply multiplied by 1000, and documented as such:

> How fragmented bluestore free space is (free extents / max possible number of free extents) * 1000

But now it's being multiplied by 1e6, making it hard to create consistent dashboards and alerts for both Reef and Squid clusters.